plugins/com.google.cloud.tools.eclipse.dataflow.ui/build.properties

```diff
@@ -4,4 +4,5 @@ bin.includes = plugin.xml,\
                META-INF/,\
                .,\
                icons/
+additional.bundles = org.eclipse.core.databinding
```

Comment:
I believe the reason is similar to https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/1490#issuecomment-282935075? (I don't have any problem without this.)